v2/emailpassword/advanced-customizations/localstorage-example/about.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: about
+title: About
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/advanced-customizations/localstorage-example/about.mdx -->
+# Localstorage instead of cookies
+
+Using the features in Advanced customization, we can customize the SDK to use localstorage instead of cookies set on the server. 
+
+You can find the complete example on [GitHub](https://github.com/supertokens/supertokens-auth-react/tree/master/examples/with-localstorage).
+
```

Comment:
Add reasons why this might be needed:
- There is an issue that talks about when cookies will not work.
- Add a caution box saying that using localstorage is not security best practice as it can lead to token theft via XSS